docs/e2e-testing.md

```diff
@@ -0,0 +1,139 @@
+  * On Windows, use the Task Manager or `taskkill /f /im chromedriver.exe` command.
+  * On OS X, use the Activity Monitor or `sudo killall chromedriver` command.
+
+### Running the tests
+- When running the test cases, a few cases may fail (this can happen due to timing issues). They can be re-run until they pass without affecting the accuracy of the tests.
+
+#### Running the tests with command line
+E2E tests follow this configuration:
+
+Test suite | Command | Results can be viewed in
+---|---|---
+`E2E tests` | `./gradlew e2eTests` | `{project folder}/build/reports/e2e-test-try-{n}/index.html`, where `{n}` is the sequence number of the test run
+Any individual E2E test | `./gradlew e2eTestTry1 --tests TestClassName` | `{project folder}/build/reports/e2e-test-try-1/index.html`
+
+
+- `E2E tests` will be run in their entirety once and the failed tests will be re-run a few times. 
+- Before running `E2E tests`, it is important to have the both front-end and back-end dev servers running locally first if you are testing against them.
+
+You can generate the coverage data with `jacocoReport` task after running tests, e.g.:
+```sh
+./gradlew e2eTests jacocoReport
+```
+The report can be found in the `build/reports/jacoco/jacocoReport/` directory.
+
+### Testing against production server
+
+If you are testing against a production server (staging server or live server), some additional tasks need to be done.
+
+1. You need to setup a `Gmail API`<sup>1</sup> as follows:
+   * [Obtain a Gmail API credentials](https://github.com/TEAMMATES/teammates-ops/blob/master/platform-guide.md) and download it.
+   * Copy the file to `src/e2e/resources/gmail-api` (create the `gmail-api` folder) of your project and rename it to `client_secret.json`.
+   * It is also possible to use the Gmail API credentials from any other Google Cloud Platform project for this purpose.
+
+1. Edit `src/e2e/resources/test.properties` as instructed is in its comments.
+   * In particular, you will need legitimate Google accounts to be used for testing.
+
+1. Login manually to TEAMMATES on the browser used for testing to add cookie with login details to the browser profile.
+   * This profile is added to the web driver so that E2E tests will start with user already logged in. 
+   * This is required as Google does not allow login by automated software.
+
+1. For Firefox, run the full test suite or any subset of it as how you would have done it in dev server. 
+   * Do note that the GAE daily quota is usually not enough to run the full test suite, in particular for accounts with no billing enabled.
+   
+1. For Chrome, you may have to run tests one at a time as multiple ChromeDriver instances cannot be opened with the same user data.
+
+<sup>1</sup> This setup is necessary because our test suite uses the Gmail API to access Gmail accounts used for testing (these accounts are specified in `test.properties`) to confirm that those accounts receive the expected emails from TEAMMATES.
```

Comment:
Again, since now only one singular account is used:

to access the Gmail account~s~ used for testing (the~se~ account~s~ ~are~ is specified in `test.properties`) to confirm that th~os~e account~s~ receives the expected emails from TEAMMATES